tgui/packages/tgui/interfaces/TelecommsMonitor.tsx

```diff
@@ -0,0 +1,157 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Stack, Input, Button, Table, LabeledList, NoticeBox } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  screen: number;
```

Comment:
Change this to a numeric enum https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums